.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "lib/suave-std"]
 	path = lib/suave-std
 	url = https://github.com/flashbots/suave-std
+[submodule "examples/web-demo/contracts/lib/suave-std"]
```

Comment:
suave-std is already linked in. should this just be a symlink instead of a second submodule?

Reply:
good catch, we can use the top-level foundry.toml config for all subdirectories!